agent/pkg/proxy/iptables.go

```diff
@@ -85,6 +87,7 @@ func (proxier *Proxier) Start() {
 		for {
 			select {
 			case <-ticker.C:
+				proxier.FlushRules()
```

Comment:
Yes, it seems a bit too violent to do so; I am wondering whether it is necessary to determine which rules are currently invalid and call DeleteRule, and the current effective rules will not repeat the EnsureRule, and the newly added part calls EnsureRule.

Reply:
Sorry, EnsureRule func When setting the route, it has been judged that it will not be set repeatedly if it already exists. So I think it is necessary to add a clearing operation for the effective records of the routing table;

Reply:
Okay, we can do this in another PR, if you are interested.

Reply:
Okay, let me see the API.